fdbclient/include/fdbclient/Tenant.h

```diff
@@ -65,41 +64,44 @@ enum class TenantState { REGISTERING, READY, REMOVING, UPDATING_CONFIGURATION, R
 // Can be used in conjunction with the other tenant states above.
 enum class TenantLockState : uint8_t { UNLOCKED, READ_ONLY, LOCKED };
 
-struct TenantMapEntry {
-	constexpr static FileIdentifier file_identifier = 12247338;
-
-	static std::string tenantStateToString(TenantState tenantState);
-	static TenantState stringToTenantState(std::string stateStr);
+std::string tenantStateToString(TenantState tenantState);
+TenantState stringToTenantState(std::string stateStr);
 
-	static std::string tenantLockStateToString(TenantLockState tenantState);
-	static TenantLockState stringToTenantLockState(std::string stateStr);
+std::string tenantLockStateToString(TenantLockState tenantState);
+TenantLockState stringToTenantLockState(std::string stateStr);
+} // namespace TenantAPI
+struct MetaclusterTenantMapEntry {
```

Comment:
Is it feasible to move the definition for this entry into one of the metacluster headers (e.g. Metacluster.h)?

Reply:
Possibly the tenant state stuff could go with it if they only apply to metaclusters